### DISCUSSION TITLE
When adding a new Vtherm all presets goes wrong

### DISCUSSION THREAD

**Discussion by slt14** [jmcollin78/versatile_thermostat#511] [Q&A] [answered]
Hello
When I add a new vtherm, all preset of configured devices goes wrong, some goes back to manual, switched off(all of them where on before).

**Comment by jmcollin78** [comment]
Hello @slt14 ,

Can you be more precise and give me some screen copy of something like that so that I will able to understand what happens.

Please first install https://github.com/jmcollin78/versatile_thermostat/releases/tag/6.3.0 which fixes bugs around configuration and let me known if your problem is still there.

**Comment by slt14** [comment, marked as answer]
Yes this issue is fixed.

Thank you Jean-Marc

**Reply by jmcollin78** [reply]
Thank you for your report !
